utility_analysis_new/tests/pre_aggregation_test.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Pre aggregation Test"""
+from absl.testing import absltest
+from absl.testing import parameterized
+
+import pipeline_dp
+import utility_analysis_new
+
+
+class PreaggregationTests(parameterized.TestCase):
+
+    @parameterized.named_parameters(
+        dict(testcase_name="Empty dataset", input=[], expected_output=[]),
+        dict(testcase_name="Small input dataset",
+             input=[
+                 ("pid0", "pk0", 1),
+                 ("pid0", "pk1", 10),
+                 ("pid0", "pk1", 5),
+             ],
+             expected_output=[('pk0', (1, 1, 2)), ('pk1', (2, 15, 2))]),
```

Comment:
Please could you add a comment describing the structure of the output pre-aggregated element?